components/TARGET_PSA/services/crypto/COMPONENT_PSA_SRV_IPC/psa_crypto_spm.c

```diff
@@ -1067,7 +1067,7 @@ psa_status_t mbedtls_psa_inject_entropy(const unsigned char *seed,
 
     handle = psa_connect(PSA_ENTROPY_ID, MINOR_VER);
     if (handle <= 0) {
-        return (PSA_ERROR_COMMUNICATION_FAILURE);
+        return (PSA_ERROR_NOT_SUPPORTED);
```

Comment:
psa_connect may fail due to resource exhaustion - does it really means that `mbedtls_psa_inject_entropy` is not supported?

Reply:
Fixed, moved the not_supported logic to the call()